src/main/scala/xerial/sbt/Sonatype.scala

```diff
@@ -76,6 +81,26 @@ object Sonatype extends AutoPlugin {
     )
   )
 
+  sealed trait ProjectHosting {
+    def homepage: String
+    def scmUrl: String
+    def developer: Developer
+    def scmInfo                     = ScmInfo(url(homepage), scmUrl)
+    def developers: List[Developer] = List(developer)
```

Comment:
As I said in the above, setting the default value here makes difficult to override this value. So we should make it empty, or totally remove this.